src/Grid/grid.jl

```diff
@@ -188,15 +183,15 @@ struct ExclusiveTopology <: AbstractTopology
 end
 
 function ExclusiveTopology(cells::Vector{C}) where C <: AbstractCell
-    cell_vertices_table = vertices.(cells) #needs generic interface for <: AbstractCell
-    vertex_cell_table = Dict{Int,Vector{Int}}() 
-    
-    for (cellid, cell_nodes) in enumerate(cell_vertices_table)
-       for node in cell_nodes
-            if haskey(vertex_cell_table, node)
-                push!(vertex_cell_table[node], cellid)
+    cell_vertex_table = vertices.(cells) #needs generic interface for <: AbstractCell
```

Comment:
why not keep the plural? you map from cell to all vertices of the cell and not to a single vertex